docker-compose.yml

```
@@ -0,0 +1,24 @@
+services:
```

É uma boa prática apresentar qual a versão do docker você está utilizando.
```suggestion
version: '3.8'

services:
```

---

pelo oque pesquisei a partir da v2 do docker compose o uso do version foi descontinuado, então não seria tão util o uso da version

mas nesse caso vou adicionar, mais por fins de aprendizado